### ISSUE TITLE
Biometric (fingerprint) lock on notes?

### ISSUE THREAD

**Issue by BloodRaven0** [SimpleMobileTools/Simple-Notes#552]
Hey, I remember in the past I was able to lock my notes with a fingerprint, but that is no longer the case - only pattern and pin are available.

In which version was this functionality removed and why? I'd like to downgrade the app version so I have this option again, can you tell me the last version of the app that offers it?

I'm running android 10.

**Comment by tibbi** [comment]
do you have fingerprint protection set up in your system?

**Comment by BloodRaven0** [comment]
@tibbi Yes, and other apps have no trouble using it.

**Comment by tibbi** [comment]
what app version are you using?

**Comment by BloodRaven0** [comment]
@tibbi  6.13.0, downloaded from F-Droid

**Comment by tibbi** [comment]
locking of notes has already been added

**Comment by BloodRaven0** [comment]
@tibbi just for the record, I'm still unable to select fingerprint as a locking option on the latest version.

**Comment by tibbi** [comment]
do you have it enabled in some system locking? Like locking your device etc

**Comment by BloodRaven0** [comment]
@tibbi Yes, as I said all other apps have no trouble using it and I'm using it to unlock my phone too.

**Comment by tibbi** [comment]
maybe the issue will be with Android 10, it works just fine on newer versions. But I honestly dont really want to spend time implementing features for older devices that will be used by always less people.

**Comment by BloodRaven0** [comment]
Fair enough.
